x-pack/plugins/observability/public/pages/slo_edit/constants.ts

```diff
@@ -35,7 +53,8 @@ export const SLO_EDIT_FORM_DEFAULT_VALUES: CreateSLOParams = {
     },
   },
   timeWindow: {
-    duration: TIMEWINDOW_OPTIONS[0].value as any, // Get this to be a proper Duration
+    duration:
```

Comment:
💬 Now that we use `CreateSLOInput`, the types are only primitives, so no more casting as any necessary